changelog/fragments/add-deafult-olm-version.yaml

```diff
@@ -0,0 +1,7 @@
+entries:
+  - description: >
+      Include a default version of OLM manifests with SDK. Add Makefile target to enable users to 
+      modify default OLM version and update manifests accordingly.
+    kind: "addition"
+    # Is this a breaking change?
+    breaking: true
```

Comment:
It is missing the steps that should be used in the migration guide since it is a breaking change.